jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py

```diff
@@ -29,8 +29,7 @@
 es_index_psap = 'cloud-governance-cost-explorer-psap'
 es_index_perf_scale = 'cloud-governance-cost-explorer-perf-scale'
 es_index_global = 'cloud-governance-cost-explorer-global'
-
-cost_tags = ['ChargeType', 'User', 'Budget', 'Project', 'Manager', 'Owner', 'LaunchTime', 'Name', 'Email', 'Environment']
+cost_tags = ['PurchaseType', 'ChargeType', 'User', 'Budget', 'Project', 'Manager', 'Owner', 'LaunchTime', 'Name', 'Email', 'Environment']
```

Comment:
Why we need a tag for PurchaseType, is it not like ec2 type data ?

Reply:
No, it represents the EC2 cost plans

Reply:
Why we need to tag it on every ec2 resource ?

Reply:
No, it's not the tag. It is the cost explorer filter dimension.

Reply:
Ok